index.css

```diff
@@ -0,0 +1,371 @@
+body{
+    margin: 0;
+    padding: 0;
+    font-family: 'Montserrat', sans-serif ;
+    background-color: #FFF5F0;
+}
+
+.header{
+   
+    padding-left: 10%;
+    padding-right: 5%;
+    height: 100%;
+    
+}
+.header__content{
+    display: flex;
+    justify-content: space-between;
+    margin: 0;
+}
+.header__item{
+    width: 100%;
+    margin: auto;
+}
+.header__logo{
+ font-weight: 700;
+ font-size: 24px;
+ line-height: 32px;
+}
+.header__nav-item{
+    display: inline;
+    text-align: left;
+    margin-right: 20px;
+    font-weight: 700;
+    font-size: 14px;
+    list-style: 24px;
+    color: #737373;
+}
+.header__item--middle{
+    text-align: left;
+}
+.header__item--right{
+    text-align: right;
+}
+.header__divButton{
```

Comment:
You should change the name of the element `divButton` doesn't make sense, no one will know what that button does

Also, try to use the same naming convention in all your codes because above you used the kebab case, and then now you're using the camel case